src/Operation/Order/OrderCreate.php

```diff
@@ -203,9 +197,6 @@ public function getVariables()
         $buyer = $this->getCustomer();
         if ($buyer !== null) {
             $array = array_merge($array, [
-                'firstname' => $buyer->getFirstName(),
-                'lastname' => $buyer->getLastName(),
-                'email' => $buyer->getEmail(),
                 'marketingPermission' => $buyer->getMarketingPermission()
```

Comment:
Don't you need to remove this too?

Reply:
Hmm, I thought this would break email features, but I guess we'll fetch via tagging